code/game/machinery/turnstile.dm

```diff
@@ -0,0 +1,87 @@
+/obj/machinery/turnstile
+	name = "turnstile"
+	desc = "A mechanical door that permits one-way access and prevents tailgating."
+	icon = 'icons/obj/turnstile.dmi'
+	icon_state = "turnstile_map"
+	power_channel = ENVIRON
+	density = FALSE
+	armor = list(melee = 50, bullet = 50, laser = 50, energy = 50, bomb = 10, bio = 100, rad = 100, fire = 90, acid = 70)
+	anchored = TRUE
+	use_power = FALSE
+	idle_power_usage = 2
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF

```

Comment:
`INDESTRUCTIBLE`

This is absolutely something that shouldn't be indestructible.

Not even sure this flag does anything on this codebase.

Reply:
Probably does otherwise it would have errored on compile, but will remove